server/index.ts

```diff
@@ -56,18 +75,39 @@ export const configSchema = schema.object({
     keepalive: schema.boolean({ defaultValue: true }),
   }),
   auth: schema.object({
-    type: schema.string({
-      defaultValue: '',
-      validate(value) {
-        if (
-          !['', 'basicauth', 'jwt', 'openid', 'saml', 'proxy', 'kerberos', 'proxycache'].includes(
-            value
-          )
-        ) {
-          return `allowed auth.type are ['', 'basicauth', 'jwt', 'openid', 'saml', 'proxy', 'kerberos', 'proxycache']`;
-        }
-      },
-    }),
+    type: schema.oneOf(
+      [
+        schema.arrayOf(schema.string(), {
+          defaultValue: [''],
+          validate(value: string[]) {
+            if (!value || value.length === 0) {
+              return `Authentication type is not configurred properly.`;
+            }
+
+            if (value.length > 1) {
+              const basicValidate = value.find((element) => {
+                return element.toLowerCase() === 'basicauth';
+              });
+
+              if (!basicValidate) {
```

Comment:
I assume this has been renamed as you marked the other comment as resolved.

Reply:
Addressed.